chipsec/cfg/pch_3xx.xml

```diff
@@ -71,13 +71,20 @@ XML configuration file for the 300 series PCH
     </register>
 
     <!-- Power Management Controller -->
-    <register name="ABASE" type="pcicfg" bus="0" dev="0x1f" fun="2" offset="0x40" size="4" desc="ACPI Base Address">
-      <field name="STYPE" bit="0" size="1" desc="Space Type (always 1 - I/O space)"/>
-      <field name="BA"    bit="8" size="8" desc="Base Address"/>
+     <register name="ABASE" type="mm_msgbus" port="0x88" offset="0x27B4" size="4" desc="ACPI Base Address">
+      <field name="MASK" bit="18" size="6" desc="MASK for lower 6 bits"/>
+      <field name="BA"    bit="2" size="14" desc="Base Address"/>
+    </register>
+    <register name="PMBASE" type="mm_msgbus" port="0x88" offset="0x27AC" size="4" desc="PM Base Address">
+      <field name="BA" bit="0" size="16" desc="Base Address" />
+      <field name="BAMRL" bit="16" size="16" desc="Memory Range Limit"/>
     </register>
     <register name="PWRMBASE" type="pcicfg" bus="0" dev="0x1f" fun="2" offset="0x10" size="8" desc="PM Base Address">
       <field name="STYPE" bit="0"  size="1"  desc="Space Type (always 0 - memory space)"/>
-      <field name="BA"    bit="13" size="51" desc="Base Address"/>
+      <field name="BA"    bit="12" size="52" desc="Base Address"/>
+      <!-- bit 31 on 3xx-OP is reserved and should be set to 0
+           BA starts at 12 on 3xx-OP and 13 on because 12 will be 0 on 3xx moving the
+           starting address to 12 -->
     </register>
     <register name="GEN_PMCON_1" type="mmio" bar="PWRMBASE" offset="0x1020" size="4" desc="General PM Configuration A">

```

Comment:
If I'm not mistaken, the `SMI_LOCK` field changes for both 3xx and 3xx-op. Here it's defined as bit 8, while in earlier PCHs it's bit 4. Could you verify this please?